cc/toolchains/gcc_arm_embedded/BUILD.bazel

```diff
@@ -0,0 +1,239 @@
+load(":config.bzl", "config")
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+filegroup(name = "empty")
+
+filegroup(
+    name = "wrappers",
+    srcs = glob([
+        "wrappers/**",
+    ]),
+)
+
+filegroup(
+    name = "ar_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:ar",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "as_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:as",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "compiler_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:bin",
+        "@gcc_arm_embedded_toolchain//:include",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "linker_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:ar",
+        "@gcc_arm_embedded_toolchain//:gcc",
+        "@gcc_arm_embedded_toolchain//:ld",
+        "@gcc_arm_embedded_toolchain//:lib",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "objcopy_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:objcopy",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "strip_files",
+    srcs = [
+        ":wrappers",
+        "@gcc_arm_embedded_toolchain//:strip",
+    ],
+    tags = ["manual"],
+)
+
+filegroup(
+    name = "all_files",
+    srcs = [
+        ":compiler_files",
+        ":linker_files",
+        "@gcc_arm_embedded_toolchain//:bin",
+    ],
+    tags = ["manual"],
+)
+
+constraint_value(
+    name = "cortex-m4",
+    constraint_setting = "@platforms//cpu",
+)
+
+constraint_value(
+    name = "cortex-m7",
+    constraint_setting = "@platforms//cpu",
+)
+
+constraint_value(
+    name = "cortex-m3",
+    constraint_setting = "@platforms//cpu",
+)
+
+constraint_value(
+    name = "cortex-a7",
+    constraint_setting = "@platforms//cpu",
+)
+
+constraint_value(
+    name = "cortex-a9",
+    constraint_setting = "@platforms//cpu",
+)
+
+constraint_value(
+    name = "unspecified_arm",
+    constraint_setting = "@platforms//cpu",
+)
+
+selects.config_setting_group(
+    name = "gcc_arm_cpu",
+    match_any = [
+        ":cortex-m4",
+        ":cortex-m7",
+        ":cortex-m3",
+        ":cortex-a7",
+        ":cortex-a9",
+        ":unspecified_arm",
+    ],
+)
+
+platform(
+    name = "cortex_m4_none_eabi",
+    constraint_values = [
+        "@platforms//os:none",
+        ":cortex-m4",
+    ],
+)
+
+platform(
+    name = "cortex_m7_none_eabi",
+    constraint_values = [
+        "@platforms//os:none",
+        ":cortex-m7",
+    ],
+)
+
+platform(
+    name = "cortex_m3_none_eabi",
```

Comment:
should we move these to the `platforms/` folder?

Reply:
Done